_data/people/richa.yml

```diff
@@ -0,0 +1,12 @@
+active: true
+focus-area:
+  - Particle Physics
```

Comment:
this needs to be in one of ia,ssl,ssc,doma,as,osglhc (see iris-hep.org/xxx.html to see what each of the abbreviations mean)

Reply:
since Sudhir asked you, it's probably ssc (training activities)

Reply:
I changed it to ssc